server/base/src/main/java/org/apache/accumulo/server/master/LiveTServerSet.java

```diff
@@ -120,11 +120,17 @@ public TabletServerStatus getTableMap(boolean usePooledConnection) throws TExcep
       if (usePooledConnection == true)
         throw new UnsupportedOperationException();
 
+      long start = System.currentTimeMillis();
```

Comment:
Ok, it appears to be more accurate however it is in general slower depending on how the nanoTime is implemented (hardware dependent).  Not too sure we really need that time of accuracy here.....